### ISSUE TITLE
React.createElement: type is invalid

### ISSUE THREAD

**Issue by DevDwarf** [FortAwesome/react-fontawesome#76]
This is my error -> `React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components)`

At the top of my file i have the import `import FontAwesomeIcon from "@fortawesome/react-fontawesome";`

and in another element i have `<FontAwesomeIcon icon="coffee" />`

**Comment by DevDwarf** [comment]
Also `node_modules/@fortawesome/fontawesome-free-solid/faCoffee"' has no exported member 'faCoffee'.` tried both `faCoffee` and `{ faCoffee }`

**Comment by rrdelaney** [comment]
I was having this problem. Generated code was outputting:

```js
import * as ReactFontawesome from '@fortawesome/react-fontawesome'
```

Here's a repro on StackBlitz: https://stackblitz.com/edit/react-f6g5fn?file=index.js

**Comment by robmadole** [comment]
@rrdelaney if you use `import ReactFontawesome from '@fortawesome/react-fontawesome'` you should be good-to-go

**Comment by rrdelaney** [comment]
Yep 😄 I got it working after finding out what was generated, I figured I should drop a repro by in case the original reporter was having the same issue

**Comment by robmadole** [comment]
Ah, cool. I see. Thanks for replying.
